components/admin/requests/guardian-information/Form.tsx

```diff
@@ -153,15 +156,31 @@ export default function GuardianInformationForm({
             <Text as="h3" textStyle="heading" paddingBottom="20px">
               {'Upload POA File'}
             </Text>
-            {/* TODO: Implement the file upload functionality using the fileList prop */}
+            {/* TODO: explicitly prevent non pdfs or files > 5Mb? */}
```

Comment:
We should be able to add this restriction within the file upload component. Please see https://stackoverflow.com/a/49490014. If it's a quick fix, it would be great to just add it in in this PR.

We can also restrict the file extension within the file upload component: https://stackoverflow.com/questions/4328947/limit-file-format-when-using-input-type-file